scripts/install_via_conda.sh

```diff
@@ -30,7 +30,7 @@ fi
 
 if [[ $PYTORCH_NIGHTLY == true ]]; then
   # install CPU version for much smaller download
-  conda install -y -c pytorch pytorch-nightly-cpu
+  conda install -y pytorch cpuonly -c pytorch-nightly
```

Comment:
unrelated to this change, but maybe we should rename install_via_conda to something like circle_ci_test_via_conda, in case users try to execute the script.

Reply:
Sounds good, yeah that's a good point! I think this originated from copying Botorch's structure, but will change naming in a separate PR.